maven-embedder/src/main/java/org/apache/maven/cli/CLIManager.java

```diff
@@ -149,6 +151,7 @@ public CLIManager()
         options.addOption( Option.builder( LEGACY_LOCAL_REPOSITORY ).longOpt( "legacy-local-repository" ).desc( "Use Maven 2 Legacy Local Repository behaviour, ie no use of _remote.repositories. Can also be activated by using -Dmaven.legacyLocalRepo=true" ).build() );
         options.addOption( Option.builder( BUILDER ).longOpt( "builder" ).hasArg().desc( "The id of the build strategy to use" ).build() );
         options.addOption( Option.builder( NO_TRANSFER_PROGRESS ).longOpt( "no-transfer-progress" ).desc( "Do not display transfer progress when downloading or uploading" ).build() );
+        options.addOption( Option.builder().longOpt( COLOR ).hasArg().desc( "Color support mode (auto/always/never)" ).build() );
```

Comment:
Out of curiosity, why only a "long" option? IIRC, this would be the first switch that only has a "long" version and no shorthand.

Reply:
If it's because both `-c` and `-C` are already taken, we could also go for `-cm` and `--color-mode`

Reply:
After discussing offline with @rfscholte and @mthmulders , I would prefer `--color-mode` anyway. `--color` suggests that you can put in a color there (as in 'red' on 'green').

Reply:
I urge to retain `--color` because many many other tools use the same option.

Reply:
I agree with @michael-o , the goal of this PR is to make things easier for users, rather than looking for a non trivial `-Dstyle.color=always` option, so the `--color` option similar to the gnu `ls` or ` grep` seems a better choice to me.

Reply:
Exactly, when I reviewed the PR for `-Dstyle.color` I requested to apply the `grep` logic to no to invent new terms.

Reply:
Ah, I was not aware of this. Then `--color` makes sense to me. 

Also just checked to be sure, the help text looks good even without a short character:
![image](https://user-images.githubusercontent.com/10269942/117192179-b3c1c700-ade1-11eb-8d2f-f274d3d1126b.png)

Reply:
Also, just checked how the help output looks without having a shorthand flag. Looks good to me:
![image](https://user-images.githubusercontent.com/10269942/117192527-20d55c80-ade2-11eb-8498-8cdd2b3c5673.png)